komodo-relational/src/test/java/org/komodo/relational/workspace/WorkspaceManagerTest.java

```diff
@@ -883,26 +900,26 @@ public void shouldExportVdb() throws Exception {
         TestUtilities.compareFileContents(cmpFile, vdbDestFile);
     }
 
-    @Test
-    public void shouldImportVdb() throws Exception {
-        String tmpDirPath = System.getProperty("java.io.tmpdir");
-        File tmpDir = new File(tmpDirPath);
-
-        long timestamp = System.currentTimeMillis();
-        myFileDir = new File(tmpDir, "myfile-" + timestamp);
-        assertTrue(myFileDir.mkdir());
-
-        File vdbSrcFile = new File(myFileDir, "vdbFile.xml");
-        String sampleExample = TestUtilities.streamToString(TestUtilities.sampleExample());
-        FileUtils.write(sampleExample.getBytes(), vdbSrcFile);
-        assertTrue(vdbSrcFile.exists());
-
-        Properties parameters = new Properties();
-        parameters.setProperty("files-home-path-property", myFileDir.getAbsolutePath());
-
-        KomodoObject parent = _repo.komodoWorkspace(getTransaction());
-        wsMgr.importVdb(getTransaction(), parent, vdbSrcFile.getName(), "file", parameters);
-
-        assertTrue(parent.hasChild(getTransaction(), TestUtilities.SAMPLE_VDB_NAME));
-    }
```

Comment:
Yeah so I hadn't committed it to the final version. I imagine you fetched a copy from an earlier version of my commit on my fork. Either way, no point in committing now since I have changed it substantially while working on the import routine.